### ISSUE TITLE
Split pane is not working

### ISSUE THREAD

**Issue by kumarom** [microsoft/terminal#3877]
# Environment

```none
Windows build number: Microsoft Windows [Version 10.0.18363.476] Professional Edition
Windows Terminal version (if applicable): Version: 0.7.3382.0

Any other software?
```

# Steps to reproduce

I have two tabs open (1st cmd, 2nd wsl1 Ubuntu) and I wanted to have split pane. So I focused on wsl1 and then pressed ALT + SHIFT + - or ALT + SHIFT + '+'. But there was no change.

Then I closed all tabs. Changed key bindings ALT + SHIFT + - to CTRL + SHIFT + h after editing profiles.json. Reopened all tabs, and then tried CTRL + SHIFT + h. Still no action.

# Expected behavior

As per release notes for v0.7.3291.0, it should have split pane.

# Actual behavior

Split pane is not working

**Comment by bluuurgh** [comment]
For me it only works if I don't use the numpad plus and min keys (which I would prefer) but it does work with the other plus and min keys (near backspace).

**Comment by Grisu118** [comment]
You used the correct key specifier for numbad keys? The following works for me:
```json
            {
                "command": "splitHorizontal",
                "keys": [
                    "alt+shift+numpad_minus"
                ]
            },
            {
                "command": "splitVertical",
                "keys": [
                    "alt+shift+numpad_plus"
                ]
            }
```

See also here: https://github.com/microsoft/terminal/issues/3856

**Comment by kumarom** [comment]
@Grisu118 Your suggetions work. However, the defaults mentioned in https://github.com/microsoft/terminal/blob/9145903e118bfa3b48749694689aec3a191bc4ac/src/cascadia/TerminalApp/defaults.json#L237-L238 do not. This issue can be closed if the defaults are updated.

**Comment by zadjii-msft** [comment]
@kumarom This is definitely weird. You're saying that 

```json
{
    "command": "splitVertical",
    "keys": [ "alt+shift+numpad_plus" ]
}
``` 

worked for you, but 


```json
{
    "command": "splitVertical",
    "keys": [ "ctrl+shift+h" ]
}
``` 

_didn't_? That's really weird. What keyboard layout are you using?

**Comment by kumarom** [comment]
@zadjii-msft All I am saying that 
```
{
    "command": "splitVertical",
    "keys": [ "alt+shift+numpad_plus" ]
}
```
worked. But `{ "command": { "action": "splitPane", "split": "vertical"}, "keys": [ "alt+shift+plus" ] },` does not work and is documented in https://github.com/microsoft/terminal/blob/9145903e118bfa3b48749694689aec3a191bc4ac/src/cascadia/TerminalApp/defaults.json#L237-L238

**Comment by zadjii-msft** [comment]
@kumarom okay, I think I got confused by your OP where you mentioned that "CTRL + SHIFT + h" didn't work.

What keyboard layout are you using?

Does the <kbd>alt+shift+plus</kbd> binding work for you if you use the <kbd>+</kbd> key that's _not_ on the numpad? On a QWERTY EN-US keyboard, this is the key with both `+` and `=`. 

Perhaps we need to update the documentation to be clearer that `plus` and `numpad_plus` are _not_ interchangable.

**Comment by kumarom** [comment]
@zadjii-msft I tried below and none of them worked for me. I am using QWERTY EN-US keyboard.
- `{ "command": { "action": "splitPane", "split": "vertical"}, "keys": [ "alt+shift+plus" ] },`
- `{ "command": { "action": "splitPane", "split": "vertical"}, "keys": [ "alt+shift+numpad_plus" ] },`
- `{ "command": { "action": "splitPane", "split": "vertical"}, "keys": [ "alt+shift++" ] },`
- `{ "command": { "action": "splitPane", "split": "vertical"}, "keys": [ "alt+shift+h" ] },`
As per defaults defined [here](https://github.com/microsoft/terminal/blob/9145903e118bfa3b48749694689aec3a191bc4ac/src/cascadia/TerminalApp/defaults.json), above should have worked.

Below works:
- `{ "command": "splitVertical", "keys": [ "alt+shift+numpad_plus" ] },`
- `{ "command": "splitVertical", "keys": [ "alt+shift++" ] },`
- `{ "command": "splitVertical", "keys": [ "alt+shift+plus" ] },`
- `{ "command": "splitVertical", "keys": [ "alt+shift+h" ] },`

**Comment by zadjii-msft** [comment]
Oh well this is horrifying. That style of keybindings definitely should be working. Lemme dig in a bit...

**Comment by zadjii-msft** [comment]
I see what's going on here.

#3722 added support for `splitPane`, but unfortunately was not included in the 0.7.3382.0 release. So You'll need to keep using `splitVertical` and `splitHorizontal` style keybindings until the next update is released. 

Sorry for any confusion!

**Comment by andreamaruccia** [comment]
My personal preference was to confiugre it like this:

* use the keypads -/+
* use splitmode so that it will select the current window's shell (if ubuntu it duplicates ubuntu, if powershell it duplicates psh) 
* works on the current version of vscode as opposed of old commands

```
        {
          "command": {
            "action": "splitPane",
            "split": "horizontal",
            "splitMode": "duplicate"
          },
          "keys": [
            "alt+shift+numpad_minus"
          ]
        },
        {
          "command": {
            "action": "splitPane",
            "split": "vertical",
            "splitMode": "duplicate"
          },
          "keys": [
            "alt+shift+numpad_plus"
          ]
        }
```
